fastscanner/services/scanners/lib/parabolic.py

```diff
@@ -102,3 +103,68 @@ async def scan(
         df = df[(df[cum_low.column_name()] - df[C.LOW]).abs() < 0.0001]
 
         return df
+
+
+class ATRParabolicUpScanner:
+    def __init__(
+        self,
+        min_adv: float,
+        min_adr: float,
+        atr_multiplier: float,
+        min_volume: float,
```

Comment:
I also realized that we need to include end_time in the scanner. Check the ParabolicDown for reference.